src/components/Reset.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import { Button } from '@chakra-ui/react';
+import axios from 'axios';
+import TaskContext from '../context/TaskContext';
+
+const Reset = () => {
+    const { setTasks } = useContext(TaskContext);
+
+    const handleReset = async () => {
+        try {
+            const response = await axios.get('http://localhost:3000/tasks');
```

Comment:
Instead of making an API call to reset. Consider making a separate new variable for tasksEdited that are being deleted/edited. So that, you can have a tasks variable which has all the tasks and a separate variable for tasks being modified.